src/main/java/com/iota/iri/conf/TipSelConfig.java

```diff
@@ -17,6 +17,11 @@
      */
     double getAlpha();
 
+    /**
+     * @return Descriptions#TIP_SELECTION_TIMEOUT
```

Comment:
```suggestion
     * @return {@value Descriptions#TIP_SELECTION_TIMEOUT}
```

We have #1159 that supposed to fix this in other configs